seacrowd/sea_datasets/sleukrith_ocr/sleukrith_ocr.py

```diff
@@ -0,0 +1,263 @@
+}
+"""
+
+_DATASETNAME = "sleukrith_ocr"
+
+_DESCRIPTION = """\
+SleukRith Set is the first dataset specifically created for Khmer palm leaf
+manuscripts. The dataset consists of annotated data from 657 pages of digitized
+palm leaf manuscripts which are selected arbitrarily from a large collection of
+existing and also recently digitized images. The dataset contains three types of
+data: isolated characters, words, and lines. Each type of data is annotated with
+the ground truth information which is very useful for evaluating and serving as
+a training set for common document analysis tasks such as character/text
+recognition, word/line segmentation, and word spotting.
+"""
+
+_HOMEPAGE = "https://github.com/donavaly/SleukRith-Set"
+
+_LANGUAGES = ["khm"]
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    "sleukrith-set": {
+        "images": "https://drive.google.com/uc?export=download&id=19JIxAjjXWuJ7mEyUl5-xRr2B8uOb-GKk",  # 1GB
+        "annotated-data": "https://drive.google.com/uc?export=download&id=1Xi5ucRUb1e9TUU-nv2rCUYv2ANVsXYDk",  # 11.7MB
+    },
+    "isolated-characters": {
+        "images_train": "https://drive.google.com/uc?export=download&id=1KXf5937l-Xu_sXsGPuQOgFt4zRaXlSJ5",  # 249MB
+        "images_test": "https://drive.google.com/uc?export=download&id=1KSt5AiRIilRryh9GBcxyUUhnbiScdQ-9",  # 199MB
+        "labels_train": "https://drive.google.com/uc?export=download&id=1IbmLg-4l-3BtRhprDWWvZjCp7lqap0Z-",  # 442KB
+        "labels_test": "https://drive.google.com/uc?export=download&id=1GYcaUInkxtuuQps-qA38u-4zxK7HgrAB",  # 354KB
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.OPTICAL_CHARACTER_RECOGNITION]
+_SEACROWD_SCHEMA = f"seacrowd_{TASK_TO_SCHEMA[_SUPPORTED_TASKS[0]].lower()}"  # imtext
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class SleukRithSet(datasets.GeneratorBasedBuilder):
+    """Annotated OCR dataset from 657 pages of digitized Khmer palm leaf manuscripts."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{_SEACROWD_SCHEMA}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=_SEACROWD_SCHEMA,
+            subset_id=_DATASETNAME,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "image_path": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=[i for i in range(111)]),
```

Comment:
Does this dataset have label metadata for each integer value from 0-110?

Reply:
No. Please refer to the discussion in the datasheet, especially Q1.

Reply:
okay, thanks for pointing it out. prob we can note that the label meta wasn't given as inline comment before L136